pkg/util/commands_test.go

```diff
@@ -108,25 +103,20 @@ func TestRunFailWithTimeout(t *testing.T) {
 	assert.Equal(t, "", res)
 	assert.Equal(t, true, cmd.DidError())
 	assert.Equal(t, true, cmd.DidFail())
-
-	os.Remove(exPath + "/" + tmpFileName)
-
 }
 
 func TestRunThreadSafety(t *testing.T) {
+	tests.SkipForWindows(t, "Pre-existing test. Windows doesn't have a decent sleep builtin to run no-interactively")
```

Comment:
```
C:\Users\jnord>timeout /?

TIMEOUT [/T] timeout [/NOBREAK]

Description:
    This utility accepts a timeout parameter to wait for the specified
    time period (in seconds) or until any key is pressed. It also
    accepts a parameter to ignore the key press.

Parameter List:
    /T        timeout       Specifies the number of seconds to wait.
                            Valid range is -1 to 99999 seconds.

    /NOBREAK                Ignore key presses and wait specified time.

    /?                      Displays this help message.

NOTE: A timeout value of -1 means to wait indefinitely for a key press.

Examples:
    TIMEOUT /?
    TIMEOUT /T 10
    TIMEOUT /T 300 /NOBREAK
    TIMEOUT /T -1
```

Reply:
Unfortunately, if you try that as part of the test the command fails saying somethnig like "it needs to be run interactively" :-(

Reply:
`failed to run 'timeout 1' command in directory 'C:\Users\Steve\code\jx\pkg\util\test_data\scripts', output: 'ERROR: Input redirection is not supported, exiting the process immediately.'`

Reply:
apparently the workaround is
`start /wait timeout 5`

Reply:
`start` is not an executable, but a command prompt builtin command.
```
         Error:      Received unexpected error:
                    exec: "start": executable file not found in %PATH%
                    failed to run 'start /wait timeout 5' command in
directory 'C:\Users\Steve\code\jx\jx\pkg\util/test_data/scripts', output: ''
```